Mathlib/CategoryTheory/Monoidal/Category.lean

```diff
@@ -348,6 +348,20 @@ theorem inv_whiskerLeft (X : C) {Y Z : C} (f : Y ⟶ Z) [IsIso f] :
     inv (X ◁ f) = X ◁ inv f := by
   aesop_cat
 
+@[simp]
+lemma whiskerLeftIso_refl (W X : C) :
+    whiskerLeftIso W (Iso.refl X) = Iso.refl (W ⊗ X) :=
+  Iso.ext (whiskerLeft_id W X)
+
+@[simp]
+lemma whiskerLeftIso_trans (W : C) {X Y Z : C} (f : X ≅ Y) (g : Y ≅ Z) :
+    whiskerLeftIso W (f ≪≫ g) = whiskerLeftIso W f ≪≫ whiskerLeftIso W g :=
+  Iso.ext (whiskerLeft_comp W f.hom g.hom)
+
+@[simp]
```

Comment:
Do we need a simp in this direction? The opposite seems more likely to me.

Reply:
The fact that that's inconsistent with the direction of the simp for refl, trans doesn't feel right to me

Reply:
I found the similar lemma `inv_whiskerLeft` has the opposite direction `inv (X ◁ f) = X ◁ inv f`, which is based on the idea of pushing non-primitive components of monoidal categories inside as much as possible. The identity and the compositions are considered as primitive, but not for `inv`. This has an advantage as we may expect that 

1. `inv f` is further simplified,
2. some lemmas involving the left whiskering can be applied to an expression containing `X ◁ inv f`, but not to `inv (X ◁ f)`.

I think this idea will also work in the case of `Iso`.

Reply:
Good points. At least like this `whiskerLeftIso_refl` is consistent with `whiskerLeft_id`, `whiskerLeftIso_trans` is consistent with `whiskerLeft_comp`, and `whiskerLeftIso_symm` is consistent with `whiskerLeft_inv`. I've changed it to be as you suggested